aplib.net-demo/Assets/Scripts/Tiles/Tile.cs

```diff
@@ -7,16 +7,21 @@ namespace Assets.Scripts.Tiles
     /// </summary>
     public abstract class Tile
     {
+        /// <summary>
+        /// The rotation of the tile. 0 = 0 degrees, 1 = 90 degrees, 2 = 180 degrees, 3 = 270 degrees.
+        /// </summary>
+        public int Rotation = 0;
+        
         /// <summary>
         /// The allowed directions for this tile. The index of the list corresponds to the direction. 0 = North, 1 = East, 2 = South, 3 = West.
         /// </summary>
-        protected List<bool> _allowedDirections;
+        protected List<bool> AllowedDirections;
```

Comment:
I'm assuming this change is correct, but where in the editor config is this? I genuinely can't find it

Reply:
Rider was yelling at me that I needed to change it to this but I don't know where the rule is defined in the editor config.